### PR TITLE
[REF] very minor tidy up

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
This just brings the 3 lines that built this 'array within an array' together & makes them a little more
readable

Before
----------------------------------------
Code confusing

After
----------------------------------------
Still confusing 

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Dedupe/Merger.php [hunk 1]
```diff
@@ -1091,8 +1091,11 @@ public static function getRowsElementsAndInfo($mainId, $otherId, $checkPermissio
         // CRM-15681 don't display sub-types in UI
         continue;
       }
-      $rows["move_$field"]['main'] = self::getFieldValueAndLabel($field, $main)['label'];
-      $rows["move_$field"]['other'] = self::getFieldValueAndLabel($field, $other)['label'];
+      $rows["move_$field"] = [
+        'main' => self::getFieldValueAndLabel($field, $main)['label'],
+        'other' => self::getFieldValueAndLabel($field, $other)['label'],
+        'title' => $fields[$field]['title']
```
Comment (yashodha) [inline CRM/Dedupe/Merger.php]: @eileenmcnaughton A comma should follow the last multiline array item.

Comment (eileenmcnaughton) [reply CRM/Dedupe/Merger.php]: dang!

**Comment by yashodha** [comment]
@eileenmcnaughton merging this

**Comment by eileenmcnaughton** [comment]
thanks!

**Pull request merged**